install_travis.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Fail if any command fails.
+set -e
+
+# Install frontend dependencies
+frontend() {
+    export CXX=clang++
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash
+    nvm install 6
+    nvm use 6
+    npm install -g gulp npm@3.10.7
+    chmod +x ./frontend.sh
+    ./frontend.sh test
+}
+
+# Install backend dependencies
+backend() {
+    pip install -r requirements/travis.txt
+    touch cfgov/static_built/js/routes/common.js
```

Comment:
Minor request: could you add a comment here, something like [this](https://github.com/cfpb/cfgov-refresh/compare/travis-faster-backend#diff-a21fa67a0ea8b49d72120a14a91a30ebR46), so it's clear why this line is here?